issue #2670 - Feature to remove thumbnail dots

marvinl803 opened the pull request:
### Summary:
This pull request works on issue #2670  removes the dots from the thumbnails.

Please review and let me know if any additional changes are required.

Before:
![issue-1](https://github.com/user-attachments/assets/1d59852a-f148-452d-b1af-75b5dd9275d8)


After:
![issue-2](https://github.com/user-attachments/assets/d774312d-acdb-4d01-ad94-dfd219242c2a)

ImprovedTube commented:
thank you again!

- for the name?:     " ⫶ " (more actions)

- why did you use JS here unlike in the other PR?  
 - then we can also make it depend on ImprovedTube.storage ... (Our CSS variables are convenient,  but conditional JS can be more CPU efficient, since not everybody will use the feature and CSS rules may be evaluated /processed from right to left.)

marvinl803 commented:
Hi,

I have made the requested changes. Initially, I attempted to implement the solution the same way I did in my previous PR, but it wasn't working as expected. After investigating, I found the issue and have now implemented the feature in the same way as my last PR. It should work correctly now.

Additionally, I updated the name as requested. Please let me know if it's correct or if you would like me to make further adjustments.

Thank you!

HornyPrivateGamer commented:
> ### Summary:
> This pull request works on issue #2670 removes the dots from the thumbnails.
> 
> Please review and let me know if any additional changes are required.
> 
> Before: ![issue-1](https://private-user-images.githubusercontent.com/27875562/386755927-1d59852a-f148-452d-b1af-75b5dd9275d8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE4MDI1MTUsIm5iZiI6MTczMTgwMjIxNSwicGF0aCI6Ii8yNzg3NTU2Mi8zODY3NTU5MjctMWQ1OTg1MmEtZjE0OC00NTJkLWIxYWYtNzViNWRkOTI3NWQ4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTE3VDAwMTAxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYzZjVjYTI4YTI3OGNiMjE3MDY2NWNkOTg2ZmI1YTk0ODk5YWRkMjBjNWUwODA4ODE2ODQyMzE3ZmUxY2JlOWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.3EwtOD0yGrgX4htnA92-oVgbrp-5SOKPGfk3ZtwPEuU)
> 
> After: ![issue-2](https://private-user-images.githubusercontent.com/27875562/386755973-d774312d-acdb-4d01-ad94-dfd219242c2a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE4MDI1MTUsIm5iZiI6MTczMTgwMjIxNSwicGF0aCI6Ii8yNzg3NTU2Mi8zODY3NTU5NzMtZDc3NDMxMmQtYWNkYi00ZDAxLWFkOTQtZGZkMjE5MjQyYzJhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTE3VDAwMTAxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZjMDlmOTBlZjRmZTRkNDg1NDYxZmJhY2RjZTZlZDJkY2Y2ODNhNWRlMzdkMjk4YTI0MDMzMjQyYmJlOTAyZTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.MuBN5FQ2xb_qDGvDPpCxHkFvHCHEYeMBUyRClglLLig)

Hi any way you could remove those margins. That there is a big empy space after removing the three dots.
Thansk

ImprovedTube commented:
> have now implemented the feature in the same way as my last PR. It should work correctly now.

thanks, that's consistent for now

>  we can also make it depend on ImprovedTube.storage ... (Our CSS variables are convenient, but conditional JS can be more CPU efficient, since not everybody will use the feature and CSS rules may be evaluated /processed from right to left.)

, while ultimately we can reduce CSS load by moving most CSS to JS, depending `if (ImprovedTube.storage...)`'s    as mentioned (didn't benchmark exactly)  ( but that's beyond #2678  )

ImprovedTube commented:
> have now implemented the feature in the same way as my last PR. It should work correctly now.

thanks

>  we can also make it depend on ImprovedTube.storage ... (Our CSS variables are convenient, but conditional JS can be more CPU efficient, since not everybody will use the feature and CSS rules may be evaluated /processed from right to left.)

, while ultimately we can reduce CSS load by moving most CSS to JS, depending `if (ImprovedTube.storage...)`'s    as mentioned (didn't benchmark exactly)  ( but that's beyond #2678  )

HornyPrivateGamer commented:
> @HornyPrivateGamer I will look to see if I can come up with a solution for those whitespaces and I will make a new PR with those changes.

ok thanks a lot, you the best

ImprovedTube commented:
> , while ultimately we can reduce CSS load by moving most CSS to JS, depending if (ImprovedTube.storage...)'s as mentioned (didn't benchmark exactly) ( but that's beyond https://github.com/code-charity/youtube/pull/2678 )

...or, in other words, next to our layout features, we can make a collection of all relevant uBlock origin rules (1000's which need no update or are updated by their authors or have a million users all updating manually yet), so then the main work remaining about this will be naming / explaining each optionally and maybe crowd-sourcing.
